sklearnex/cluster/dbscan.py

```diff
@@ -98,7 +98,7 @@ def _onedal_fit(self, X, y, sample_weight=None, queue=None):
         self._onedal_estimator.fit(X, y=y, sample_weight=sample_weight, queue=queue)
         self._save_attributes()
 
-    def _onedal_cpu_supported(self, method_name, *data):
+    def _onedal_supported(self, method_name, *data):

```

Comment:
Patching verbosity should be added.

Reply:
Please read the description to the draft pr properly.